### PR TITLE
Cleanup some of the stuff

### PR DESCRIPTION
**Pull request by welchyd**
Removes unused elements and provides some general cleanup.

### CODE REVIEW

File: src/main/xsd/state/1.0.0/state.xsd [hunk 1]
```diff
@@ -1136,20 +1069,7 @@
...
-        Not sure the analyze element should be here,
-        maybe a simple name/value extension point?
-      -->
-      <xs:element name="analyze" type="xs:boolean" minOccurs="0"
-        maxOccurs="1">
-        <xs:annotation>
-          <xs:documentation xml:lang="en" xml:space="preserve">
-             Flag to indicate if this layer can be used
-              for data analytics.
+            Refersh rate for the layer.
```
Comment (schmidtk) [inline src/main/xsd/state/1.0.0/state.xsd]: Could you fix the typo here please?

File: src/main/xsd/state/1.0.0/state.xsd [hunk 2]
```diff
@@ -1237,7 +1157,7 @@
         maxOccurs="1">
         <xs:annotation>
           <xs:documentation xml:lang="en" xml:space="preserve">
-            Optional original URL reference.
+            Optional orginal URL reference.
```
Comment (schmidtk) [inline src/main/xsd/state/1.0.0/state.xsd]: Fix spelling of "original".

File: src/main/xsd/state/1.0.0/state.xsd [hunk 3]
```diff
@@ -2160,7 +2071,7 @@
         minOccurs="0" maxOccurs="1">
         <xs:annotation>
           <xs:documentation xml:lang="en" xml:space="preserve">
-           Flag that the original (unconverted) value is stored.
+           Flag that the orginal (uncoverted) value is stored.
```
Comment (schmidtk) [inline src/main/xsd/state/1.0.0/state.xsd]: Fix spelling of "original".

File: src/main/xsd/state/1.0.0/state.xsd [hunk 4]
```diff
@@ -2413,12 +2324,12 @@
   <xs:simpleType name="FloatType">
     <xs:annotation>
       <xs:documentation xml:lang="en" xml:space="preserve">
-        Specalized float data type that also allows "Infinity"
+        Specalized float data type that also allows "Infinity".
```
Comment (schmidtk) [inline src/main/xsd/state/1.0.0/state.xsd]: Fix spelling of "specialized".

File: src/main/xsd/state/1.0.0/state.xsd [hunk 5]
```diff
@@ -2430,7 +2341,7 @@
   <xs:simpleType name="IntegerType">
     <xs:annotation>
       <xs:documentation xml:lang="en" xml:space="preserve">
-        Specalized integer data type that also allows "Infinity"
+        Specalized integer data type that also allows "Infinity".
```
Comment (schmidtk) [inline src/main/xsd/state/1.0.0/state.xsd]: Fix spelling of "specialized".

File: src/main/xsd/state/1.0.0/state.xsd [hunk 6]
```diff
@@ -2447,7 +2358,7 @@
    <xs:simpleType name="PositiveIntegerType">
     <xs:annotation>
       <xs:documentation xml:lang="en" xml:space="preserve">
-        Specalized positive integer data type that also allows "Infinity"
+        Specalized positive integer data type that also allows "Infinity".
```
Comment (schmidtk) [inline src/main/xsd/state/1.0.0/state.xsd]: Fix spelling of "specialized".

**Comment by welchyd** [comment]
Nice, looks like someone else fixed many of the typos while I was working on this.
I found another one not mentioned ... hopefully that does it though

**Pull request merged**